app/src/pages/organization/organizationProjectsPage/projectsListTable/projectsListTable.jsx

```diff
@@ -129,28 +144,52 @@ export const ProjectsListTable = ({ projects, sortingDirection, onChangeSorting
       <BubblesLoader />
     </div>
   ) : (
-    <Table
-      data={data}
-      primaryColumn={primaryColumn}
-      fixedColumns={fixedColumns}
-      sortingDirection={sortingDirection.toLowerCase()}
-      sortingColumn={primaryColumn}
-      sortableColumns={primaryColumn.key}
-      rowActionMenu={rowActionMenu}
-      className={cx('projects-list-table')}
-      onChangeSorting={onTableColumnSort}
-    />
+    <div className={cx('with-pagination-container')}>
+      <Table
+        data={data}
+        primaryColumn={primaryColumn}
+        fixedColumns={fixedColumns}
+        sortingDirection={sortingDirection.toLowerCase()}
+        sortingColumn={primaryColumn}
+        sortableColumns={primaryColumn.key}
+        rowActionMenu={rowActionMenu}
+        className={cx('projects-list-table')}
+        onChangeSorting={onTableColumnSort}
+      />
+      {pageCount > 1 && (
```

Comment:
Why such a condition?
In this case the user will not be able to change even page size to see more pages.

Reply:
It's correct
totalPages={pageCount} 
If there is only one page then we do not show pagination

Reply:
Done.